nncf/openvino/rt_info.py

```diff
@@ -17,6 +17,22 @@
 from nncf.scopes import IgnoredScope
 
 
+def handle_ignored_scope(value: IgnoredScope) -> IgnoredScope:
+    """
+    Dump Ignored Scope parameters more gracefully.
+    https://github.com/openvinotoolkit/nncf/issues/2564
+    Remove keys where value is empty and key is `validate`
+
+    :param value: IgnoredScope instance, based on the quantization parameters
+    """
+    value.__dict__.pop("validate")
```

Comment:
I'd say that it's not expected from the user standpoint that after passing parameters to `dump_parameters` the parameters are modified. Please use a copy of `IgnoredScope`

Reply:
In this, I created a copy of `ignored_scope` and then made the changes but still, when it goes into `model.set_rt_info` as it dumps the value, it also updates the original data. That newly copied `ignored_scope` value is passed `set_rt_info` here.

Reply:
Could you elaborate this `when it goes into model.set_rt_info as it dumps the value, it also updates the original data`?
Do you mean that still with your latest changes `ignored_scope` got changed on dumping parameters?

Reply:
Yes. This is why this [test](https://github.com/openvinotoolkit/nncf/actions/runs/8344527642/job/22869598068?pr=2578) is failing. Since [here](https://github.com/Viditagarwal7479/nncf/blob/3024c889b36de846339200942ecd9cc36f671a23/nncf/openvino/rt_info.py#L62) whatever the value is passed into the `model.set_rt_info` function gets dumped finally and gets updated at the model side which `model.get_rt_info` retrieves.